
<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<meta http-equiv="X-UA-Compatible" content="IE=edge">
	<meta name="viewport" content="width=device-width, initial-scale=1">
	<!-- The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags -->
	<meta name="description" content="M WANG Lab Website">
	<meta name="author" content="Qi Li">
	<link rel="icon" href="./graphics/icon.jpg">

	<title>M WANG Lab</title>

	<!-- Bootstrap core CSS -->
	<link href="./css/bootstrap.min.css" rel="stylesheet">

	<!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
	<link href="./css/ie10-viewport-bug-workaround.css" rel="stylesheet">

	<!-- Custom styles for this template -->
	<link href="./css/carousel.css" rel="stylesheet">
	
	<!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
	<!--[if lt IE 9]>
	<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
	<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
	<![endif]-->
	<style>
.high{ image-resolution:300dpi;}
.low{ image-resolution:0dpi;}
.divcss5-c p{margin-left: 2em;margin:3px 0}
</style>
</head>
<!-- NAVBAR
================================================== -->
<body>
	<div class="navbar-wrapper">
	<div class="container">

		<nav class="navbar navbar-inverse navbar-static-top">
		<div class="container">
			<div class="navbar-header">
			<button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false" aria-controls="navbar">
				<span class="sr-only">Toggle navigation</span>
				<span class="icon-bar"></span>
				<span class="icon-bar"></span>
				<span class="icon-bar"></span>
			</button>
			<a href="#" class="navbar-left"><img src="./graphics/icon.jpg" width="50px" height="50px"></a>
			<a class="navbar-brand" href="#"></a>
			</div>
			<div id="navbar" class="navbar-collapse collapse">
			<ul class="nav navbar-nav">
				<li><a href="index.html">Home</a></li>
				<li class="dropdown"><a href="research.html">Research</a>
				<div class="dropdown-content">
					<p><a href="human.html">Human genetics</a></p>
					<p><a href="virus.html">Virus genetics</a></p>
					<p><a href="othermethods.html">Other methods</a></p>
				</div>
				<li><a href="people.html">People</a></li>
				<li><a href="teaching.html">Teaching</a></li>
				<li><a href="contact.html">Join Us</a></li>
			</ul>
			</div><!--/.navbar-collapse -->
		</div>
		</nav>

	</div>
	</div>

	<!-- Carousel
	================================================== -->
	<div id="myCarousel" class="carousel slide" data-ride="carousel">
	<!-- Indicators -->
	<ol class="carousel-indicators">
		<li data-target="#myCarousel" data-slide-to="0" class="active"></li>
		<li data-target="#myCarousel" data-slide-to="1"></li>
		<li data-target="#myCarousel" data-slide-to="2"></li>
	</ol>
	<div class="carousel-inner" role="listbox">
		<div class="item active">
		<img class="first-slide" src="./graphics/carousel/cover.jpg" alt="First slide">
		<div class="container">
			<div class="carousel-caption">
			<h1>M WANG Lab</h1>
			</div>
		</div>
		</div>
		<div class="item">
		<img class="second-slide" src="./graphics/carousel/human.jpg" alt="Second slide">
		<div class="container">
			<div class="carousel-caption">
			<h1>Human genetics</h1>
			</div>
		</div>
		</div>
		<div class="item">
		<img class="third-slide" src="./graphics/carousel/virus.jpg" alt="Third slide">
		<div class="container">
			<div class="carousel-caption">
			<h1>Virus genetics</h1>
			</div>
		</div>
		</div>
	</div>
	<a class="left carousel-control" href="#myCarousel" role="button" data-slide="prev">
		<span class="glyphicon glyphicon-chevron-left" aria-hidden="true"></span>
		<span class="sr-only">Previous</span>
	</a>
	<a class="right carousel-control" href="#myCarousel" role="button" data-slide="next">
		<span class="glyphicon glyphicon-chevron-right" aria-hidden="true"></span>
		<span class="sr-only">Next</span>
	</a>
	</div><!-- /.carousel -->

	<div class="container marketing">
	<!-- START THE FEATURETTES -->
	<div class="page-header" id="virus genetics">
		<h1>Virus genetics</h1>
	</div>

	<div class="row featurette">
			<h2>Vaccine effectiveness prediction<span class="text-muted"></span></h2>
		<div class="col-md-6">
		<p class="lead">
		<!-- Colored version -->
		<!-- We develop and apply statistical methods to investigate complex traits such as <span style="color: #006dcc"><em>diabetes</em></span>, <span style="color: #006dcc"><em>schizophrenia</em></span> and others.
		We have devised novel tests to increase statistical power of large scale genome wide association studies, and developed new methods to estimate <span style="color: #5cb85c"><em>polygenic risk scores</em></span>, and <span style="color: #5cb85c"><em>heritabilities</em></span> of various traits. -->
		<ul>
		<p style="font-size:18px">Traditionally, vaccine effectiveness (VE) is estimated only after the population has been vaccinated and a portion of subjects have been infected.  We establish the VE prediction method by modelling the relationship between VE and genetic distance (GD) of circulating viruses against vaccine strain, which allows immediate in silico prediction of VE before a new genetic variant enters a population.</p>
		<p style="font-size:18px">The method enables timely evaluation of existing cohort vaccine protection to facilitate public health policy making and vaccine strain selection.</p>
		<p></p>
		<li><strong>Influenza VE prediction</strong></li>
		<div class = "divcss5-c">
		<p style="font-size:16px">The initial statistical framework of VE-GD relationship is developed for influenza vaccine (Cao et al, 2021, Vaccine). One major issue in this study is determining the key mutations influencing VE, and aggregation model of the multivalent vaccine. Many of the challenge in modelling influenza vaccine haven’t emerged for the COVID-19 vaccines.</p>
		<p style= "color:rgb(95,95,191);font-size:14px">Cao L, Lou J, Zhao S, Chan RW, Chan M, Wu WK, Chong MK, Zee BC, Yeoh EK, Wong SY, Chan PK, Wang MH*. In silico prediction of influenza vaccine effectiveness by sequence analysis. Vaccine. 2021 Feb 12;39(7):1030-4.</p>
		<p style= "color:rgb(95,95,191);font-size:14px">Cao L, Zhao S, Lou J, Zheng H, Chan RW, Chong MK, Chen Z, Chan PK, Zee BC, Wang MH*. Differential Influence of Age on the Relationship between Genetic Mismatch and A (H1N1) pdm09 Vaccine Effectiveness. Viruses. 2021 Apr;13(4):619</p>
		</div>
		<p></p>
		<li><strong>COVID-19 VE prediction</strong></li>
		<div class = "divcss5-c">
		<p style="font-size:16px">Based on the primary model we established for the influenza vaccines, we expand the framework for COVID-19 VE prediction. Unlike the influenza vaccines that are mostly manufactured by egg-based manufacture platforms, the COVID-19 vaccines adopt diversified technologies, which is the main variation to control in the modelling process. Meanwhile, analysis of SARS-CoV-2 genome benefits from the abundancy of datasets and the number of VE studies. The prediction accuracy by the proposed VE-GD model on independent validation dataset reached 95%.</p>
		<p style= "color:rgb(95,95,191);font-size:14px">Cao L, Lou J, Chan SY, Zheng H, Liu C, Zhao S, Li Q, Mok KP, Chan SY, Chan RWY, Chong MKC, Wu WKK, Chen Z, Wong ELY, Chan PKY, Zee BCY, Yeoh EK, Wang MH* (2022) Rapid evaluation of COVID-19 vaccine effectiveness against symptomatic infection with SARS-CoV-2 variants by analysis of genetic distance. Nature Medicine.<a style="text-decoration:underline" target="_blank" href="https://rdcu.be/cPNdp">https://rdcu.be/cPNdp</a></p>
		</div>
		</ul>
		</p>
		</div>
		<div class="col-md-6">
		<img class="human1 featurette-image img-responsive center-block" src="./graphics/v1.png" alt="v1">
		</div>
	</div>

	<hr class="featurette-divider">

	<div class="row featurette">
		<h2>g-measure: a summary statistic for virus mutation activities<span class="text-muted"></span></h2>
		<div class="col-md-6">
		<p class="lead">
		<ul>
		<p style="font-size:18px">While the human genome collected in a population exhibits a stable landscape of mutation distributions, virus genome sampled at different time points shows dynamic mutation profiles due to fast virus evolution.</p>
		<p style="font-size:18px">Clearly, summarizing the virus mutation activities through time will provide useful information about epidemics and virus evolution. Surprisingly, such a measure wasn’t unavailable in 2010s. Therefore, we introduced the g-measure (Wang Lou et al 2021, Journal of Infection), which is simply calculated as the sum of mutation prevalence of the virus in samples collected in a given time interval t.</p>
		<p><span style="font-size:18px;">&nbsp;&nbsp;&nbsp;&nbsp;<strong>g</strong><sub>t</sub> = <strong>m</strong><sub>t</sub> · <strong>p</strong><sub>t</sub></span></p>
		<p style="font-size:18px">where m is an indicator variable (J-vector) of the key mutations over virus genome at time t, and p is the mutation prevalence in population at time t (J-vector). J is the genome size. Therefore, g summarizes overall mutation activities level in virus genome in a certain population within a given time period. The g-measure is shown to provide a good account of the population epidemic trend (see Figure in the right).</p>
		<p style="font-size:16px"><strong>Primary method:</strong></p>
		<p style= "color:rgb(95,95,191);font-size:14px">Wang MH, Lou J, Cao L, Zhao S, Chan RW, Chan PK, Chan MC, Chong MK, Wu WK, Wei Y, Zhang H, Zee BC, Yeoh EK (2021) Characterization of key amino acid substitutions and dynamics of the influenza virus H3N2 hemagglutinin. Journal of Infection. 2021 Oct</p>
		<p style= "color:rgb(95,95,191);font-size:14px">2018 US Provisional Patent No. 62/687,645, 18/MED/829, 2019 PCT/CN2019/091652 Measurement and Prediction on Influenza Virus Genetic Mutation Patterns.   Wang MH, Zee BCY, Chong KC, Lou J.</p>
		<p style="font-size:16px"><strong>Applications of the g-measure:</strong></p>
		<p style= "color:rgb(95,95,191);font-size:14px">Lou J, Zhao S, Cao L, Chong MK, Chan RW, Chan PK, Zee BC, Yeoh EK, Wang MH *. Predicting the dominant influenza A serotype by quantifying mutation activities. International Journal of Infectious Diseases. 2020 Aug 22.</p>
		<p style= "color:rgb(95,95,191);font-size:14px">Lou J, Zheng H, Zhao S, Cao L, Wong ELY, Chen Z, Chan RWY, Chong MKC, Zee BCY, Chan PKS, Yoeh EK, Wang MH* (2022) Quantifying the effect of government interventions and virus mutations on transmission advantage during COVID-19 pandemic. Journal of Infection and Public Health.</p>
		</ul>
		</p>
		</div>
		<div class="col-md-6">
		<img class="virus2 featurette-image img-responsive center-block" src="./graphics/v2.png" alt="v2">
		</div>
	</div>

	<hr class="featurette-divider">

	<div class="row featurette">
		<h2>Statistical identification of influential mutations: the effective mutations (EMs)<span class="text-muted"></span></h2>
		<div class="col-md-6">
		<p class="lead">
		<ul>
		<font style="font-size:18px">With the g-measure, we can perform "feature selection" in the virus genome, to determine which mutation sets, as a function of mutation prevalence, provide a best match to population epidemics. This leads to the identification of the <strong>Effective mutations</strong> (EMs), which is a time-dependent subset of nucleotide substitutions demonstrating statistically significant association to epidemic trends (Wang Lou et al 2021)</font>
		<p style="font-size:18px;text-indent:2em">We found that 80% of these EMs for influenza A(H3N2) HA were located  in the antigenic sites, and the rest were located in the immuno-subdominant region HA2. This suggests that the statistically determined mutation sets might provide useful molecular candidates. Indeed, the EM sites offer the “features” that are predictive to influenza vaccine effectiveness, and enabled the VE prediction framework for influenza (Cao et al 2021, Vaccine)</p>
		<p style="font-size:18px;text-indent:2em">As infectious disease pathogens evolves through time to escape from population adaptive immune responses, an once "effective" mutation that gives the pathogen selective advantage in the population will not be effective forever. Therefore, we recognize the necessity to propose the quantity"<strong>Effective mutation period</strong>(EMP)”, which captures for how long the key mutations would demonstrate advantage in the host population against its immune recognition. The EMP is jointly influenced by transmission dynamics, population density, immunity profiles in the population, and other epidemiological characteristics. The g-measure is dependent on the EM and EMP, thus it can capture the rises and falls of epidemic cycles over many seasons.</p>
		</ul>
		</p>
		</div>
		<div class="col-md-6">
		<img class="virus2 featurette-image img-responsive center-block" src="./graphics/v3.png" alt="v4">
		</div>
	</div>
	
	<hr class="featurette-divider">
	<h2>Predicting virus evolution: beth-1<span class="text-muted"></span></h2>
		<div class="row featurette">
		<div class="col-md-6">
		<p class="lead">
		<ul>
		<p style="font-size:18px"><strong>Prediction of influenza vaccine strains for the next epidemic season</strong></p>
		<p style="font-size:16px">A new method to predict influenza virus evolution and select vaccine strains for the influenza virus.</p>
		<p style= "color:rgb(95,95,191);font-size:14px">2018 US Provisional Patent No. 62/687,645, 18/MED/829, 2019 PCT/CN2019/091652 Measurement and Prediction on Influenza Virus Genetic Mutation Patterns.Wang MH, Zee BCY, Chong KC, Lou J.</p>
		<p style="font-size:18px"><strong>Estimation for evolutionary rate</strong></p>
		<p style="font-size:16px">We develop a new probability framework to estimate virus substitution rate by genomic region, fast and effective, without building phylogeny.</p>
		<br />
		<p style="font-size:16px">More to be introduced…</p>
		</ul>
		</p>
		</div>
		<div class="col-md-6">
		<img class="virus2 featurette-image img-responsive center-block" src="./graphics/v4.png" alt="v4">
		</div>
	</div>
		<hr class="featurette-divider">
		
	<h2>Codon optimization / de-optimization<span class="text-muted"></span></h2>
		<div class="row featurette">
		<div class="col-md-6">
		<p class="lead">
		<ul>
		<p style="font-size:16px">A new method to perform codon optimization or deoptimization preserving viral structure information, with applications in improving efficiency of virus replication and antigenicity for vaccine production.</p>
		<p style= "color:rgb(95,95,191);font-size:14px">2021 US Provisional No: 63/283,91: Codon de-optimization or optimization using genetic architecture, Wang MH, Zheng H, Zee BCY</p>
		<br />
		<p style="font-size:16px">More to be introduced…</p>
		</ul>
		</p>
		</div>
	</div>
	<hr class="featurette-divider">

	<!-- /END THE FEATURETTES -->


	<!-- FOOTER -->
	<footer>
		<p class="pull-right"><a href="#">Back to top</a></p>
		<p>&copy; 2022 M WANG Lab. All rights reserved.</p>
	</footer>

	</div><!-- /.container -->


	<!-- Bootstrap core JavaScript
	================================================== -->
	<!-- Placed at the end of the document so the pages load faster -->
	<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.4/jquery.min.js"></script>
	<!--<script>window.jQuery || document.write('<script src="../../assets/js/vendor/jquery.min.js"><\/script>')</script>-->
	<script src="./js/bootstrap.min.js"></script>
	<!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
	<script src="./js/ie10-viewport-bug-workaround.js"></script>
</body>
</html>
